### ISSUE TITLE
Not redirecting to install.php

### ISSUE THREAD

**Issue by ugintl** [govCMS/govcms8-installer#611]
When I try to install the site, it shows the root folder instead of taking me to the install page. I have to click "docroot" in order to start the installation. My address is appearing as /docroot/core/install.php

**Comment by thisisalistairsaccount** [comment]
Hi @ugintl are you able to work through the installation process? Is it just the redirect not functioning as expected to get it started?

**Comment by ugintl** [comment]
Yes the installation had no issues, but links are appearing like example.com/docroot/core/install.php/.......

I am facing this problem with all the distributions having docroot folder. Other distributions which have "web" folder does not have this problem. So I guess it has something to do with the "docroot" folder.

**Comment by thisisalistairsaccount** [comment]
Have you tried our upcoming Drupal 9 release as of yet? https://github.com/govCMS/GovCMS

I'm curious to see if the issue is also there - and as were also moving towards that distribution this year, I'm not sure if it's worth investigating this on this Drupal 8 distribution as we're shifting most of our work towards 9. Of course, if this issue also exists there, I think there's probably value on our end to investigate and resolve from a convenience factor.
